chandra_aca/tests/test_maude_decom.py

```diff
@@ -14,6 +14,46 @@
     test_data.update(pickle.load(f))
 
 
+def compare_tables(t1, t2, keys=None, exclude=()):
```

Comment:
There are functions for comparing tables, but I suspect they won't be quite what you want. If you asked before writing your own then it might have been worth using what's there, but since you have something working then we can stick with that.

Reply:
well, this would be used in other places (at least aca_view) so anything that helps... or we could add a ska_helper

Reply:
`testr.test_helper` would be a good place. But let's just merge this and then refactor when the need actually arises.